docs/source/contrib.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+Thank you for contributing to `osier`! All contributions, bug reports, bug fixes, documentation 
+improvements, enhancements and ideas are welcome.
+
+The [GitHub "Issues" tab](https://github.com/arfc/osier/issues) contains open issues. Some issues
+are labeled "Difficulty:1-Beginner" and are good first issues to tackle for new contributers to 
+`osier`. You can even sort by "label" to find precisely those issues!
+
+## Bug reports
+Bug reports from users are an important for identifying unforeseen issues. If you find a bug please
+open a new issue with the following:
+1. An explanation of the problem with enough details for others to reproduce the problem. Some 
+common information needed is
```

Comment:
It's not a minimum requirement because some issues might not need all that information! For example, the code might run without error, but the results don't make sense.